onboarding/urls.py

```diff
@@ -0,0 +1,31 @@
+"""
+The urls for on-boarding app.
+"""
+from django.urls import re_path
+
+from . import views
+
+
+urlpatterns = [
+    # url(r"^onboarding/recommendations/$", views.recommendations, name="recommendations"),
```

Comment:
Let's remove this and leave it as an empty list.

Reply:
File deleted